config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
+  devise_for :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root to: "home#index"
```

Comment:
This does nothing but causes an error as you have set a route with no associated home controller with an index action. If you do this you need the home controller and index action with associated `index.html.erb` view in views which would be automatically generated upon creating the controller.